vault/router.go

```diff
@@ -473,8 +473,13 @@ func (r *Router) routeCommon(req *logical.Request, existenceCheck bool) (*logica
 		resp, err := re.backend.HandleRequest(req)
 		if resp != nil &&
 			resp.Auth != nil &&
-			resp.Auth.Alias != nil {
-			resp.Auth.Alias.MountAccessor = re.mountEntry.Accessor
+			!strings.HasPrefix(req.Path, "renew") {
```

Comment:
What is the purpose of the addition of this check? When will it succeed?

Reply:
Added a comment in the code quoting the reason.

Reply:
👍